katana/repl/__init__.py

```diff
@@ -343,6 +343,14 @@ def __init__(self, manager: Manager):
         # Register hook to update prompt
         self.register_cmdfinalization_hook(self.finalization_hook)
 
+        # Set a default flag-format if none supplied
+        if self.manager.flag_pattern is None:
```

Comment:
Dumb question: shouldn't this be `self.manager.flag_format`?

Reply:
It's possible that would also work.  flag_pattern is the compiled regex of flag_format, so when flag_format is set using manager, flag_pattern is also being set.